packages/council-graphql/src/resolvers/index.ts

```diff
@@ -49,12 +50,49 @@ export const resolvers: Resolvers<CouncilContext> = {
     voter: (_, { address }) => {
       return VoterModel.getByAddress({ address });
     },
-    voters: (_, { addresses }, context) => {
+    voters: async (_, { addresses, search }, context) => {
+      let voters: (Voter | undefined | null)[] = [];
+
       if (addresses) {
-        return VoterModel.getByAddresses({ addresses });
+        voters = await VoterModel.getByAddresses({ addresses });
       } else {
-        return VoterModel.getAll({ context });
+        voters = await VoterModel.getAll({ context });
       }
+
+      if (search) {
+        const candidates: string[] = [];
+
+        voters = await Promise.all(
+          voters.map(async (voter) => {
+            if (!voter) {
+              return null;
+            }
```

Comment:
NIT: We might be able to remove the `null` type from `voters` then just return nothing on line 68.

so line 54 would be `let voters: (Voter | undefined)[]`

and line 68 would be `return;`

Reply:
good call!